package.json

```diff
@@ -18,9 +18,9 @@
   },
   "license": "BSD-2-Clause",
   "dependencies": {
-    "acorn": "^7.2.0",
+    "acorn": "^7.3.1",
     "acorn-jsx": "^5.2.0",
-    "eslint-visitor-keys": "^1.2.0"
+    "eslint-visitor-keys": "github:eslint/eslint-visitor-keys#add-chain-expression"
```

Comment:
We need to update it to a new version of `eslint-visitor-keys` after we released it.

Reply:
I just did a release of `eslint-visitor-keys`. Should be safe to update this now.

Reply:
Thank you. I updated this line.